### ISSUE TITLE
Pop up a notification when localsend is loading in the background on android

### ISSUE THREAD

**Issue by iewnfod** [localsend/localsend#1012]
### Is your feature request related to a problem? Please describe.

I used to have an iphone and a mac so I can just synchronize the clipboard and send files very easy on these two devices. But recently, I changed my phone into an android one so I cannot synchronize my clipboard or send file, until I find this application. Localsend really helps my a lot but I also want some more features to help android have a better experience. 

When I was trying to send a file or my clipboard from my macbook to my android phone, my mac will tell me that it is waiting for my phone to accept the connect to send the file. However, when I was send a file from my phone to the mac, the application can be directly called up to the front and ask my to accept the request. I think this function is really good and the android version also needs it. 

### Describe the solution you'd like

I think it would be good to send a message to the user when there is a request, just like what those message applications do. A notification can be popped up on the top when localsend is not opening at the front.

### Describe alternatives you've considered

Maybe call the application to the front directly is easier to implement. However, users' experience would be worser since it might interrupt what the user are currently doing.

**Comment by GarryOneW00d** [comment]
Yes, the option with notifications is cool, of course, if it is possible to implement it. And here is the option with output to the foreground is probably better implemented as a separate setting as desired. After all, it is necessary to test whether such a scenario will often interfere.

**Comment by EdwardWilsonJr** [comment]
Yes, the notification support is a bit lacking

> 'And here is the option with output to the foreground is probably better implemented as a separate setting as desired'

Agree!

**Comment by EdwardWilsonJr** [comment]
> Sorry to be a bit off-topic, but is it possible to disable the automatic opening of the application when transferring files on a computer? Because I did not see such an item anywhere in the settings. And this is a little disturbing, on Windows, the icon in the taskbar window flashes red if something happens to the program, and that's enough for me, so I would like to disable automatic opening.

No, there isn't such option

**Comment by silverwings15** [comment]
the Android app really needs notifications or some way to bring the app to the foreground of the phone imo..

**Comment by nathfavour** [comment]
> ### Is your feature request related to a problem? Please describe.
> I used to have an iphone and a mac so I can just synchronize the clipboard and send files very easy on these two devices. But recently, I changed my phone into an android one so I cannot synchronize my clipboard or send file, until I find this application. Localsend really helps my a lot but I also want some more features to help android have a better experience.
> 
> When I was trying to send a file or my clipboard from my macbook to my android phone, my mac will tell me that it is waiting for my phone to accept the connect to send the file. However, when I was send a file from my phone to the mac, the application can be directly called up to the front and ask my to accept the request. I think this function is really good and the android version also needs it.
> 
> ### Describe the solution you'd like
> I think it would be good to send a message to the user when there is a request, just like what those message applications do. A notification can be popped up on the top when localsend is not opening at the front.
> 
> ### Describe alternatives you've considered
> Maybe call the application to the front directly is easier to implement. However, users' experience would be worser since it might interrupt what the user are currently doing.
> 
> ### Additional context
> _No response_

This would be greatly needed

**Comment by nathfavour** [comment]
> > ### Is your feature request related to a problem? Please describe.
> > I used to have an iphone and a mac so I can just synchronize the clipboard and send files very easy on these two devices. But recently, I changed my phone into an android one so I cannot synchronize my clipboard or send file, until I find this application. Localsend really helps my a lot but I also want some more features to help android have a better experience.
> > When I was trying to send a file or my clipboard from my macbook to my android phone, my mac will tell me that it is waiting for my phone to accept the connect to send the file. However, when I was send a file from my phone to the mac, the application can be directly called up to the front and ask my to accept the request. I think this function is really good and the android version also needs it.
> > ### Describe the solution you'd like
> > I think it would be good to send a message to the user when there is a request, just like what those message applications do. A notification can be popped up on the top when localsend is not opening at the front.
> > ### Describe alternatives you've considered
> > Maybe call the application to the front directly is easier to implement. However, users' experience would be worser since it might interrupt what the user are currently doing.
> > ### Additional context
> > _No response_
> 
> This would be greatly needed

I'm ready to take up the challenge of implementing this, if necessary

**Comment by LHAlengend** [comment]
Still wait for this feature

**Comment by geolaw** [comment]
I run localsend in the background, on my swaywm scratchpad and always have to go searching for the window when trying to send between computers.  a notification popup would be so much easier.

**Comment by junkeythong** [comment]
Hi, i also want this feature, how about its status? Thanks
